app/Console/Commands/SchemaDumpCommand.php

```diff
@@ -58,7 +58,9 @@ public function handle(ConnectionResolverInterface $connections, Dispatcher $dis
         \Artisan::call('migrate', $parameters, $stdout);
 
         $file = $this->option('path') ?: base_path('/misc/db_schema.yaml');
-        $yaml = Yaml::dump(Schema::dump($database), 3, 2);
+        $dump = Schema::dump($database);
+        ksort($dump);
```

Comment:
Odd, most tables have a specific order and I wouldn't think any should deviate from that order. (Do we care about that, I'm not so sure)

Reply:
Point was to have a stable db_schema.yaml table order.

Reply:
It should already be deterministic.   Only time it differs is from people doing weird things.